grafonnext/bin/mkschema.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -euo pipefail
+IFS=$'\n\t'
+
+if [ "$#" != 2 ]; then
+  echo "Usage: $(basename "$0") VERSION OUTPUTDIR"
+  exit 1
+fi
+
+VERSION="$1"
+OUTPUTDIR="$2"
+
+mkdir -p "$OUTPUTDIR/$VERSION"
+
+TEMPDIR=$(mktemp -d)
+
+trap "rm -rf $TEMPDIR" EXIT
+
+git clone --branch "$VERSION" --depth 1 https://github.com/grafana/grafana.git "$TEMPDIR"
+cd "$TEMPDIR"
+
+cd pkg/coremodel/dashboard
+thema lineage gen jsonschema > "$OUTPUTDIR/$VERSION/dashboard.json"
+cd -
+
+# Made an assumption here, generating jsonschema for these don't work.
+#for FILE in $(find public -name models.cue); do
+#    DIR=$(dirname "$FILE")
+#    NAME=$(basename "$DIR")
+#    cd "$DIR"
+#    thema lineage gen jsonschema > "$OUTPUTDIR/$VERSION/$NAME.json"
+#    cd -
+#done
```

Comment:
@sdboyer As you can see here, I made an attempt at getting the schemas for the panels. Looking forward to the pipelining that make these available.

Reply:
Yeah, saw that. They're all assembled in [this generated file](https://github.com/grafana/grafana/blob/main/pkg/plugins/pfs/corelist/loadlist_gen.go), accessible by calling [this](https://github.com/grafana/grafana/blob/main/pkg/plugins/pfs/corelist/new.go#L18).

(not everything in that list has a `models.cue`; that's everything with a `plugin.json`/the full core plugin list)

Go code is incoming that puts this to work.

Reply:
This has been covered by the go script.